app/src/lib/components/Button.svelte

```diff
@@ -34,8 +23,8 @@
 	export let align: 'flex-start' | 'center' | 'flex-end' | 'stretch' | 'baseline' | 'auto' = 'auto';
 	export let isDropdownChild = false;
 	// Style props
-	export let style: ButtonStyle = 'neutral';
-	export let kind: ButtonKind = 'soft';
+	export let style: ColorStyle = 'neutral';
+	export let kind: KindStyle = 'soft';
```

Comment:
`KindStyle` sounds like tautology btw, i.e. both words have similar meaning. I would also argue that color and style can't be combined, it should be one or the other. Wdyt?

Reply:
I agreed but don't want to make the name too vague